src/pages/ApplyInfo/ButtonBox.js

```diff
@@ -0,0 +1,21 @@
+import { useField } from 'formik';
+
+const ButtonBox = ({ buttonData, name, text }) => {
+  const [field, meta] = useField(name);
+  const selectedValue = meta.value;
+  // const isSelected = value => (value === selectedValue ? true : false);
+  return (
+    <>
+      {text}
+      {buttonData.map(data => (
+        <div key={data}>
+          <input {...field} type="radio" value={data} />
+          <span> {data}</span>
+        </div>
+      ))}
+      {field.value && meta.touched && meta.error ? <div>{meta.error}</div> : null}
```

Comment:
조건부 렌더링일때는 굳이 삼항연산자일 필요 없습니다. 아래 suggestion 으로도 충분해요

```suggestion
      {field.value && meta.touched && meta.error && <div>{meta.error}</div>}
```